tests/test_api_build.py

```diff
@@ -18,6 +18,7 @@
 
 from binstar_client.commands import remove, show
 from binstar_client.errors import NotFound
+from pkg_resources import parse_version
```

Comment:
Are these tests getting installed with `conda-build`? If so, we need to add `setuptools` to `run` dependencies.